scaling_performance/optimizing_storage.adoc

```diff
@@ -0,0 +1,125 @@
+[[scaling-performance-optimizing-storage]]
+= Optimizing Storage
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+
+== Optimizing Storage
+
+Docker stores images and containers in a graph driver (a pluggable storage
+backend), such as Device Mapper, Overlay, and Btrfs. Each have advantages and
+disadvantages. For example, Overlay is faster than Device Mapper at starting and
+stopping containers, but is not POSIX compliant because of the architectural
+limitations of a union file system, and does not yet support SELinux.
+
+For more information about Overlay, including supportability and usage caveats,
+link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html-single/7.3_Release_Notes/index.html#technology_previews_file_systems[see
+the RHEL 7.3 Release Notes].
+
+In production environments, using a LVM thin pool on top of regular block
+devices (not loop devices) for container images and container root file systems
+storage is recommended. 
+
+[NOTE]
+====
+Using a Loop device back-end can effect performance issues. While you can still
+continue to use it, Docker logs a warning message, but you can still continue to
```

Comment:
Remove "but you can still continue to use it" in the "Docker logs..." sentence. redundant.